stylesheets/print.scss

```diff
@@ -0,0 +1,52 @@
+@page {
+  margin-top: 1.5cm;
+  margin-right: 1cm;
+  margin-left: 1cm;
+  margin-bottom: 2cm;
+}
+
+body {
+  font: 11pt Georgia, "Times New Roman", Times, serif;
+}
+
+nav, footer, div.sidebar {
```

Comment:
Given that the specification is _so long_ (40 pages when I try to print it), I think it would be quite helpful to have a table of contents at the top. This would help orient new readers to the scope of the spec, but also give readers who want to jump to a specific section a way to see roughly how far in the document their desired section is and what the sections around it are called. In others words, I'm saying I think I'd leave `div.sidebar` visible, and maybe format it to be as table of contents-y as possible.

If you wanted to go the extra mile, you could follow the headings in the `div.sidebar` table of contents with their page number! (See [here](http://stackoverflow.com/questions/13465747/how-do-i-add-a-page-number-to-a-link-to-an-anchor).) That would be incredibly useful.